### DISCUSSION TITLE
Install

### DISCUSSION THREAD

**Discussion by rddoove** [nkasco/IT-Admin-Toolkit-WinUI#16] [General]
I like what i see, but i am missing a good install guide. Can you help me with this?

**Comment by nkasco** [comment]
Hi did you check out the ReadMe?

**Reply by pax212fr** [reply]
Hi, I did download all files but it's not clear what to do to install : no exe, no ps1, no batch to launch... and yes i read .md...

**Reply by nkasco** [reply]
More/less it's a portable install. You'd run ITATKWinUI.exe. 

Make sure you install the runtime from the readme.

**Reply by pax212fr** [reply]
I must be dumb : where is that .exe ???

**Reply by nkasco** [reply]
I'm guessing you sourced the repository instead of just downloading the pre compiled zip from the releases page. 

If you just source the repo you'd have the build/compile the sln yourself.

**Reply by pax212fr** [reply]
when i'm here : https://github.com/nkasco/IT-Admin-Toolkit-WinUI i don't see any release page... what am i missing ? can u send me the link plz ?

**Reply by pax212fr** [reply]
been to "code" it's the only place where's a zip... i downloaded it but ain't no .exe...

**Reply by nkasco** [reply]
https://github.com/nkasco/IT-Admin-Toolkit-WinUI/releases/tag/v1.0.5
